test/cookies.test.js

```diff
@@ -0,0 +1,19 @@
+import { getCookieValue, setCookieValue } from '../src/cookies';
+
+beforeEach(() => {
+    document.cookie = '';
+});
+
+describe('getCookieValue()', () => {
+    it('returns a string value if there is a cookie', () => {
```

Comment:
Can you split this into two tests; one to verify that `setCookieValue` works as expected and the other that `getCookieValue` works as expected, rather than testing both together.

Reply:
I can, but I think the test code will look exactly the same